src/shardus/index.ts

```diff
@@ -2936,10 +2934,12 @@ class Shardus extends EventEmitter {
       const changeHash = this.crypto.hash(change)
       //skip handled changes
       if (this.appliedConfigChanges.has(changeHash)) {
+        activeConfigChanges.add(changeHash)
```

Comment:
the comment says //skip handled changes but now we are adding the hash to activeConfigChanges.  Is this intended?